dnsop-structured-dns-error-page.txt

```diff
@@ -357,9 +357,9 @@ Internet-Draft            Data for Filtered DNS                June 2022
 
 4.1.  Client Generating Request
 
-   When generating a DNS query, the client MUST include the OPT pseudo-
-   RR [RFC6891] to elicit the Extended DNS Error option [RFC8914] in the
-   DNS response.
+   When generating a DNS query, the client MUST include the EDE OPT
+   pseudo-RR [RFC6891] to elicit the Extended DNS Error option [RFC8914]
```

Comment:
You may want to replace "MUST include the EDE OPT pseudo-RR" with "sets the EDE option in the OPT pseudo-RR".
I don't see the need for normative language; the behavior to include EDE option in the query is implicitly defined in the EDE spec.